src/test-harness/SparkLiquidityLayerTests.sol

```diff
@@ -1995,9 +2128,16 @@ abstract contract SparkLiquidityLayerTests is SpellRunner {
         inputs[7] = "accept: application/json";
 
         string memory response = string(vm.ffi(inputs));
+        string memory message  = vm.parseJsonString(response, string(abi.encodePacked(".message")));
+
+        // If the response is not successful, sleep and try again
+        if (keccak256(abi.encodePacked(message)) == keccak256(abi.encodePacked("NOTOK"))) {
+            vm.sleep(1000);  // Prevent rate limiting from Etherscan (5 calls/second)
+            return _getEvents(chainId, target, topic0, retryCount++);
+        }
 
         // Get Result Array Length
-        uint i;
+        uint256 i;
```

Comment:
To be consistent with `for (uint256 i = 0; i < 1000; i++)`, this should probably be `uint256 i = 0;`